plugins/permission-backend-module-opa-wrapper/src/opa-client/opaClient.ts

```diff
@@ -14,6 +14,7 @@ import { LoggerService } from '@backstage/backend-plugin-api';
 export class OpaClient {
   private readonly opaEntryPoint?: string;
   private readonly opaBaseUrl?: string;
+  private readonly opaPolicyFallback?: string;
```

Comment:
We _could_ create a type for this like the below in the `types.ts`

```ts
export type OpaFallbackPolicy = 'allow' | 'deny' | undefined;
```

```ts
private readonly opaPolicyFallback?: OpaFallbackPolicy;
```

WDYT?

Reply:
I thought of doing that but I didn't see any similar ones so I didn't do it. I'll make it happen then

Reply:
https://github.com/Parsifal-M/backstage-opa-plugins/pull/220/commits/76d83684eadf78d3e7a126608c2ba5f4731b5e63

LMK if this is okay.